### PR TITLE
Fix travis

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
This takes care of having developer profiles that include all devices and both developer and distribution profiles that match the format required by travis. Also, changes from using AdHoc to AppStore for distribution.

### CODE REVIEW

File: Psorcast/Psorcast.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -893,7 +893,7 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = org.sagebionetworks.Psorcast;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = psorcastAdHoc;
+				PROVISIONING_PROFILE_SPECIFIER = "match Distribution org.sageBionetworks.psorcast";
```
Comment (zaro0508) [inline Psorcast/Psorcast.xcodeproj/project.pbxproj]: you created a distribution profiles for psorcast and psorcastValidation0?  Then you must update all fastlane files that contain profile name, for example https://github.com/Sage-Bionetworks/Psorcast-iOS/blob/master/fastlane/Matchfile#L8

it must match exactly including being case sensitive.  Also we need to add the new profiles to https://github.com/Sage-Bionetworks/ios-certificates/tree/KA9Z8R6M6K/profiles   Instructions to do that are here: https://sagebionetworks.jira.com/wiki/spaces/BRIDGE/pages/142316906/iOS+CI+Setup  look at "To export and encrypt the profile do this" section.

Comment (syoung-smallwisdom) [reply Psorcast/Psorcast.xcodeproj/project.pbxproj]: Updated the case-sensitive to match.

**Comment by zaro0508** [comment]
@syoung-smallwisdom have you tested by running a command line build? It won’t work until you add the new AppStore profiles to the iOS-certificates repo

**Comment by syoung-smallwisdom** [comment]
@zaro0508 Go look. I have added them manually to iOS-certificates.

I am merging this and if it doesn't work we can go for a round two. Perhaps you will be able to fix it at this point. This is blocking development by third-party contractors.

**Pull request merged**